tidb-cloud/manage-serverless-spend-limit.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Manage Spend Limit for Serverless Tier clusters
+summary: Learn how to manage spend limit for your Serverless Tier clusters.
+---
+
+# Manage Spend Limit for Serverless Tier Clusters
+
+> **Note:**
+>
+> The spend limit is only applicable to Serverless Tier clusters. For Dedicated Tier clusters, TiDB Cloud charges according to the resources that you consume.
+
+For each Serverless Tier cluster, TiDB Cloud provides a free usage quota as follows:
+
+- Row storage: 5 GiB
+- Column storage: 5 GiB
+- Request Units (RU): 50 MB per month
```

Comment:
Not sure MB is the correct unit for RU. See https://docs.pingcap.com/tidb/dev/tidb-resource-control#what-is-request-unit-ru